blinko/umbrel-app.yml

```
@@ -0,0 +1,32 @@
+manifestVersion: 1
+id: blinko
+name: Blinko
+tagline: An AI-powered hub for your ideas and notes
+category: files
+version: "0.38.5"
```

Update to latest version.
```suggestion
version: "0.38.6"
```